Sources/ArcGISToolkit/Components/Bookmarks/Bookmarks.swift

```diff
@@ -173,41 +170,41 @@ extension Bookmarks {
         )
     }
     
-    /// A view that shows the list of bookmarks.
-    ///
-    /// - Note: Once the minimum supported platform is 16.4 or greater, the `ScrollView`, `VStack`
-    /// and `ForEach` can be replaced with a `List` with `scrollContentBackground(.hidden)` applied.
-    private var list: some View {
-        ScrollView {
-            VStack(alignment: .leading) {
-                ForEach(sortedBookmarks, id: \.self) { bookmark in
-                    Button {
-                        selectedBookmark = bookmark
-                    } label: {
-                        Text(bookmark.name)
-                            // Make the entire row tappable.
-                            .frame(maxWidth: .infinity, alignment: .leading)
-                            .contentShape(Rectangle())
-                    }
-                    .buttonStyle(.plain)
-                    .padding(4)
-                    if bookmark != sortedBookmarks.last {
-                        Divider()
-                    }
+    /// Makes a view that shows a list of bookmarks.
+    /// - Parameter bookmarks: The bookmarks to be shown.
+    @ViewBuilder private func makeList(bookmarks: [Bookmark]) -> some View {
+        if bookmarks.isEmpty {
+            noBookmarks
+        } else {
+            List(bookmarks.sorted { $0.name <  $1.name }, id: \.self, selection: selection) { bookmark in
+                // When 'init(isPresented:bookmarks:viewpoint:)' and
+                // 'init(isPresented:geoModel:viewpoint:)' are removed, this
+                // button can be replaced with 'Text' and the list's selection
+                // mechanism and 'onChange(of: selection)' can be used instead.
+                Button {
+                    selectBookmark(bookmark)
+                } label: {
+                    Text(bookmark.name)
+                        // Make the entire row tappable.
+                        .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .leading)
+                        .contentShape(Rectangle())
                 }
+                .buttonStyle(.plain)
+#if targetEnvironment(macCatalyst)
+                .listRowBackground(bookmark == selection?.wrappedValue ? nil : Color.clear)
+#endif
             }
+            .listStyle(.plain)
         }
-        .padding([.horizontal])
     }
     
-    /// A view that is shown while a `GeoModel` is loading.
-    private var loading: some View {
-        ProgressView()
+    /// Makes a view that is shown while a `GeoModel` is loading.
+    private func makeLoadingView(with geoModel: GeoModel) -> some View {
```

Comment:
I can't tell if this is something that was actually changed in this PR or not, so I'll just comment on it and leave it up to you whether you want to do anything about it. It isn't a big deal since this is a private API, but according the the Swift API design guidelines, "The first argument to initializer and [factory methods](https://en.wikipedia.org/wiki/Factory_method_pattern) calls should not form a phrase starting with the base name." In other words, `geoModel` should not have a custom argument label (`with`).